test/metabase/api/search_test.clj

```diff
@@ -648,23 +661,23 @@
                               (:data (make-search-request :crowberto [:q search-term])))]
             (model-index/add-values! model-index)
 
+            ;; TODO This is necessary because we've disabled model-index realtime sync due to toucan issues
```

Comment:
I think we can also remove this hack thanks to the manual sync we do in the model indexing code.